entries/ad-layers-dfp/index.js

```diff
@@ -107,7 +114,7 @@ import './style.scss';
         }
       }
     }
-    return this.buildAd(args.slotName, args.path, args.sizes, args.targeting, args.sizeMapping);
+    return this.buildAd(args.slotName, args.path, args.sizes, args.targeting, args.sizeMapping, args.companion);
```

Comment:
This companion will need some fallback value (the others are set above)
```suggestion
    return this.buildAd(args.slotName, args.path, args.sizes, args.targeting, args.sizeMapping, args.companion || false);
```